src/diffusers/pipelines/controlnet/pipeline_controlnet_sd_xl.py

```diff
@@ -672,6 +672,7 @@ def __call__(
         width: Optional[int] = None,
         num_inference_steps: int = 50,
         guidance_scale: float = 5.0,
+        use_dynamic_scale: bool = False,
```

Comment:
To allow for more flexibility for future changes could we maybe add a parameter
```suggestion
        guidance_scale_type="static"  # dynamic,
```